src/instructlab/model/list.py

```diff
@@ -29,6 +29,7 @@
 )
 @click.option(
     "--list-checkpoints",
+    help="List the trained model checkpoints.",
```

Comment:
You should probably reword this to "Also include trained model checkpoints" or something.

Reply:
So it's listing both trained model checkpoints and existing models?

Reply:
Yeah it's not a binary flag, it's mainly a fallback case we check if the given path isn't a safetensors directory or gguf model:

https://github.com/instructlab/instructlab/blob/f0590b710689d0ae0051e744b47f2517b563a828/src/instructlab/model/list.py#L97

Reply:
How is this?

Reply:
I think the way you've written it is good for a command description, but we'll probably want to change it to "include the trained checkpoints in the output", since we're extending the existing functionality.